.vscode/launch.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
Did you mean to include this file in here?

Reply:
it's fine